x/axelarnet/module.go

```diff
@@ -395,3 +400,25 @@ func extractTokenFromAckOrTimeoutPacket(packet channeltypes.Packet) sdk.Coin {
 
 	return sdk.NewCoin(trace.IBCDenom(), amount)
 }
+
+// Temporary logic to handle in-transit IBC transfers during upgrade. Previously IBC transfers sent from asset escrow address; now sent from IBC account.
+// Deprecated: Remove this function after the v1.1 upgrade
+func refund_from_asset_escrow_address_to_ibc_account(ctx sdk.Context, packet channeltypes.Packet, bank types.BankKeeper) error {
+	// Packet is validated by the IBC module, so we can safely assume it's a valid ICS20 packet
+	data := funcs.Must(types.ToICS20Packet(packet))
+
+	// decode the sender address
+	originalSender := funcs.Must(sdk.AccAddressFromBech32(data.Sender))
+	if originalSender.Equals(types.AxelarIBCAccount) {
+		return nil
+	}
+
+	// parse the denomination from the full denom path
+	trace := ibctransfertypes.ParseDenomTrace(data.Denom)
+
+	// parse the transfer amount
+	transferAmount := funcs.MustOk(sdk.NewIntFromString(data.Amount))
```

Comment:
```suggestion
	originalSender := funcs.Must(sdk.AccAddressFromBech32(data.Sender))
	if originalSender.Equals(types.AxelarIBCAccount) {
		return nil
	}

	denom := ibctransfertypes.ParseDenomTrace(data.Denom).IBCDenom()
	transferAmount := funcs.MustOk(sdk.NewIntFromString(data.Amount))
```
comments are mostly self evident from the code?